packages/fluentui/accessibility/src/behaviors/Dropdown/dropdownSelectedItemBehavior.ts

```diff
@@ -10,11 +10,11 @@ import { Accessibility } from '../../types';
 export const dropdownSelectedItemBehavior: Accessibility<DropdownSelectedItemBehaviorProps> = props => ({
   attributes: {
     root: {
-      'aria-selected': true,
```

Comment:
Should we really remove this attr?

Reply:
I think yes we should remove. Now it has role option and adding there 'aria-selected' doesn't make sense, because we are not really doing selection on this item. Only we focus the item and user can delete it. 
Even the role listbox/option is not the ideal, but we cannot find any better solution, how to switch off screen reader mode for JAWS and NVDA.